minitest/repositores/name_repository.py

```
@@ -0,0 +1,77 @@
+from .base_repository import BaseRepository
+from ..tmp_database import tmp_user_db
```

Ruff (0.8.2)

`..tmp_database.tmp_user_db` imported but unused

Remove unused import: `..tmp_database.tmp_user_db`

(F401)